aiohttp/_websocket/reader_py.py

```diff
@@ -319,10 +321,14 @@ def _feed_data(self, data: bytes) -> None:
 
     def parse_frame(
         self, buf: bytes
-    ) -> List[Tuple[bool, Optional[int], Union[bytes, bytearray], Optional[bool]]]:
+    ) -> List[
+        Tuple[bool, Optional[int], Union[bytes, bytearray, memoryview], Optional[bool]]
```

Comment:
Maybe worth making an alias of something here. The union atleast appears to be repeated in several places.